blueocean-core-js/src/js/capability/CapabilityAugmenter.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * Created by cmeyers on 9/8/16.
+ */
+import capable from './Capable';
+
+const addClass = (clazz, classMap) => {
+    const className = clazz._class;
+
+    if (!classMap[className]) {
+        // eslint-disable-next-line no-param-reassign
+        classMap[className] = [];
+    }
+
+    classMap[className].push(clazz);
+};
+
+const canWalk = (item) => item && (typeof item === 'object' || Array.isArray(item));
+
+const DEFAULT_IGNORED_PROPS = ['_links'];
```

Comment:
NIT: would add any other known things like `$pager`, though it shouldn't be fetching through the pagination fetch... but still